model_correction:
  type: "multiple_manipulators"
  multiple_manipulators:
    -
      type: "UavWpManipulator"
      indexes:
        start: 0
        end: 10
      t_body_manipulator:
        translation: [0, 0, 0.075]
        rotation: [1.57079632679, 0, 3.14159265359]
      manipulator_dof: 5
    - 
      type: "UavWpManipulator"
      indexes:
        start: 11
        end: 21
      t_body_manipulator:
        translation: [0, 0, 0.075]
        rotation: [1.57079632679, 0, 3.14159265359]
      manipulator_dof: 5